src/parser/warlock/affliction/modules/features/Darkglare.js

```diff
@@ -74,18 +63,11 @@ class Darkglare extends Analyzer {
   constructor(...args) {
     super(...args);
     // if player has Absolute Corruption, disregard the Corruption duration (it's permanent debuff then)
+    this._dotDurations = getDotDurations(this.selectedCombatant.hasTalent(SPELLS.CREEPING_DEATH_TALENT.id));
     this._hasAC = this.selectedCombatant.hasTalent(SPELLS.ABSOLUTE_CORRUPTION_TALENT.id);
```

Comment:
Might want to move the comment down here since _dotDurations doesn't have anything to do with AC?